Formula/gum.rb

```diff
@@ -0,0 +1,33 @@
+class Gum < Formula
+  desc "Tool for glamorous shell scripts"
+  homepage "https://github.com/charmbracelet/gum"
+  url "https://github.com/charmbracelet/gum/archive/v0.1.0.tar.gz"
+  sha256 "f89228867cd61ebeae0b7bcc9e8034079da256e2c6bf696a9eceb73fc4c3f2b6"
+  license "MIT"
+  head "https://github.com/charmbracelet/gum.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w -X main.Version=#{version}", "-trimpath", "-o", bin/name
+
+    man_page = Utils.safe_popen_read(bin/"gum", "man")
+    (man1/"gum.1").write man_page
+
+    bash_output = Utils.safe_popen_read(bin/"gum", "completion", "bash")
+    (bash_completion/"gum").write bash_output
+
+    zsh_output = Utils.safe_popen_read(bin/"gum", "completion", "zsh")
+    (zsh_completion/"_gum").write zsh_output
+
+    fish_output = Utils.safe_popen_read(bin/"gum", "completion", "fish")
+    (fish_completion/"gum.fish").write fish_output
+  end
+
+  test do
+    assert_equal "  Gum\n", shell_output("#{bin}/gum format 'Gum'").chomp
```

Comment:
This is a nitpick, but is it guaranteed that `gum format 'Gum'` will always begin with two spaces, even in later versions? (and end with a newline)
If not, it might be worth using `assert_match "Gum", …` instead of `assert_equal`.

Reply:
Ah, that is a very good point. Thank you!

Reply:
Fixed!

Reply:
As for the other tests, I think they would work equally well as either `asset_equal` or `assert_match`, as long as no other characters (mostly likely whitespace) have a chance of appearing in the output.

Reply:
Yep, I agree. I think the other tests work well with `assert_equal` as there is somewhat of a guarantee that they will not add whitespace in the future, so it is important to test for this.

Reply:
Thank you so much for your help with getting this PR into shape ❤️

Reply:
You're welcome! I'm glad I could help.